### PR TITLE
fix(tslint): correctly pad the script content

### PR DESCRIPTION
**Pull request by backbone87**
this avoids manipulating the tslint ruleset, which is only necessary
because the padding of vueCompiler does not work for lang="ts" scripts.
this also makes vue-cli tslint work with tslint-plugin-prettier

prereq for: https://github.com/vuejs/vue-cli/issues/761
unexpected upstream behavior:
https://github.com/vuejs/vue/blob/dev/src/sfc/parser.js#L119

### CODE REVIEW

**Comment by backbone87** [comment]
notes on leading and trailing linebreaks inside script content:

first leading linebreak (imediately after the script tag) is preserved:
```
<script>
import Vue from 'vue';
</script>
```
-> will become
```
<script>
import Vue from 'vue';
</script>
```

no leading linebreak is preserved:
```
<script>import Vue from 'vue';
</script>
```
-> will become
```
<script>import Vue from 'vue';
</script>
```


trailing linebreak is added, if not present (after linter fixing)
```
<script>
import Vue from 'vue';</script>
```
-> will become
```
<script>
import Vue from 'vue';
</script>
```

therefore this is not possible:
```
<script>import Vue from 'vue';</script>
```
-> will become
```
<script>import Vue from 'vue';
</script>
```

**Comment by backbone87** [comment]
after experimenting more with the single line scripts, i noticed that vscode + vetur dont correctly syntax color them anyway, so i dropped the preservation of single line scripts completely.

there is just one of caveat:
`<script>import Vue from 'vue';`: no leading linebreak in script content wont be reported as a lint failure (at least for prettier, i guess its the some for consecutive newlines tslint rule). however when the file is written (because of another lint failure), this leadine linebreak is added.`

so basically with prettier the following would be reported as error:
```
<script>

import Vue from 'vue';
```
while
```
<script>
import Vue from 'vue';
```
and
```
<script>import Vue from 'vue';
```
causes no error.
if the file gets written (because of any lint error in the file), all result in the follow:
```
<script>
import Vue from 'vue';
```

this could be solved by creating a custom tslint rule for vue files, where this special case is handled

**Comment by backbone87** [comment]
any update on this? its really annoying not being able to use the lint command especially in CI

**Comment by sodatea** [comment]
Thanks for the PR.

As TSLint has been [officially deprecated](https://github.com/palantir/tslint/issues/4534), we do not plan to add any new feature to this functionality and it will be removed in the next major.
So I'm closing this PR.

Thanks anyway!
